templates/single-cleanbc-projects.html

```diff
@@ -0,0 +1,18 @@
+<!-- wp:template-part {"slug":"header","theme":"bcgov-wordpress-block-theme","tagName":"header","lock":{"move":true,"remove":true},"className":"site-header"} /-->
```

Comment:
I worry that this is polluting the core of the theme to be site agnostic.... Just observation.

Reply:
Agreed, I will pull the project specific CPT and we'll either work around it or come up with an alternate solution at a later date.